app/bundles/EmailBundle/Controller/EmailController.php

```diff
@@ -1319,6 +1321,92 @@ public function batchDeleteAction(Request $request)
         );
     }
 
+    public function scheduleSendAction(CorePermissions $security, EmailModel $model, Request $request, int $objectId): JsonResponse|Response
```

Comment:
Please, write a functional test for this new action

Reply:
@kuzmany see request here